phf_builder/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+//! DEPRECATED: removed in 0.8.0.
 #![doc(html_root_url="https://docs.rs/phf_builder/0.7")]
```

Comment:
By the way, is this necessary? This seems like it would break local docs rendering by forcing all links to point to docs.rs.

Reply:
No, the root URL is only used if you don't build the docs for that crate locally. (i.e. it makes `cargo doc --no-deps` work).